src/tests/api/Filter.test.js

```diff
@@ -0,0 +1,180 @@
+import { buildFeatureFilter } from 'src/api/Filter';
+
+const filters = {
+  column1: {
+    in: {
+      owner: 'widgetId1',
+      values: ['a', 'b', 'c']
+    }
+  },
+  column2: {
+    between: {
+      owner: 'widgetId2',
+      values: [[1, 2, 3]]
+    }
+  }
+};
+
+const makeFeatureWithValueInColum = (value, column = 'column1') => [
+  {
+    type: 'Feature',
+    geometry: {
+      type: 'Point',
+      coordinates: [0, 0]
+    },
+    properties: {
+      [column]: value
+    }
+  },
+  {
+    [column]: value
+  }
+];
+
+describe('Filters', () => {
+  test('should return 1 if no filters present', () => {
+    const params = { filters: {}, type: 'number' };
+    expect(buildFeatureFilter(params)()).toBe(1);
+  });
+
+  test('should return true if no filters present', () => {
+    const params = { filters: {}, type: 'boolean' };
+    expect(buildFeatureFilter(params)()).toBe(true);
+  });
+
+  describe('feature passes filter - boolean type', () => {
+    const params = { filters, type: 'boolean' };
+
+    describe('should return false if feature column value is falsy', () => {
+      const columnValues = [0, null, undefined, false, ''];
+      for (const tc of columnValues) {
+        test(`${tc} - with geojson feature`, () => {
+          const [geojsonFeature] = makeFeatureWithValueInColum(tc);
+          const withProps = buildFeatureFilter(params)(geojsonFeature);
```

Comment:
I would change "withProps" to "isFeatureIncluded" or similar... From my view "withProps" is related to the feature, not the result of the filter